alchemist/src/orchid/resources/wiki/usage/indoor.md

```diff
@@ -4,4 +4,16 @@ title: "Simulation of indoor environments"
 
 ---
 
+In order to load the map of the indoor environment, you can use the {{ anchor('ImageEnvironment') }}.
 
+ImageEnvironment loads the map as raster image from file, interpreting the black pixels as obstacles
+(areas that cannot be accessed by nodes like wall). 
+The color of the pixel that represent obstacle can be set as constructor's parameter of the environment.
+
+In this example the image with the map is in theclasspath in the folder `images`:
+```yaml
+incarnation: protelis
+environment:
+  type: ImageEnvironment
+  parameters: [images/foo.png]
+```
```

Comment:
Can you add a few details on network models? Namely: how to configure connections in such a way that they do not get across walls

Also, a snapshot may be of help to better understand how it works.
Ideally, also an example yaml file that makes nodes move with a brownian move in an environment with obstacles

Reply:
resolved